### PR TITLE
Add support for mssql connector

### LINKED ISSUE #156: Support for mssql://
Add support for the `sql_server.pyodbc` engine, which dj-database-url supports.

### PR DESCRIPTION
**Pull request by Jafnee**
closes #156

### CODE REVIEW

**Comment by Jafnee** [comment]
Hi @joke2k , are there any changes required on the PR?

**Comment by beruic** [comment]
I need this too. Is there anything we can do to have this merged soon?

**Comment by beruic** [comment]
I need this too. @joke2k is there anything we can do to have this merged soon?

**Comment by joke2k** [comment]
Yep, i'll release soon as a possible.

File: environ/test.py [hunk 1]
```diff
@@ -215,6 +217,17 @@ def test_db_url_value(self):
         self.assertEqual(sqlite_config['ENGINE'], 'django.db.backends.sqlite3')
         self.assertEqual(sqlite_config['NAME'], '/full/path/to/your/database/file.sqlite')
 
+        sqlite_config = self.env.db('DATABASE_MSSQL_URL')
```
Comment (Jafnee) [inline environ/test.py]: Whoops! my mistake, just realized I didn't change the variable name

**Comment by Jafnee** [comment]
@joke2k I've fixed https://github.com/joke2k/django-environ/pull/157#discussion_r170956381 by amending the commit

**Comment by Jafnee** [comment]
Actually I've realized that this isn't useful. Because it's the same as using `pyodbc://`

@beruic Would using that work instead for you?

**Comment by beruic** [comment]
So I can basically just use `pyodbc://` instead of `mssql://`?

For me it is also a question of customers understanding of things, so if `pyodbc://` works, I'd love a `mssql://` alias anyway.

**Comment by Jafnee** [comment]
Correct, as long as you want to use ODBC to connect to your SQL server.

According to info from Microsoft https://docs.microsoft.com/en-us/sql/connect/python/python-driver-for-sql-server#getting-started there are 2 drivers for Python, pyodbc and pymssql.

> There are several python SQL drivers available. However, Microsoft places its testing efforts and its confidence in pyodbc driver.

Just from that pyodbc should be the choice for `mssql://`

**Comment by beruic** [comment]
I agree

**Comment by Jafnee** [comment]
In that case, this PR already does what we agree on

**Comment by joke2k** [comment]
I add just the alias

**Pull request closed without merging**